### PR TITLE
Add ellipsis to Truncate & TruncateWords

### LINKED ISSUE #59: truncate filter doest add ellipsis by defulat
Using the truncate filter in a OC blog summary list doesn't add the ellipsis by default as specified by the liquid language reference:
https://help.shopify.com/themes/liquid/filters/string-filters#truncate

### PR DESCRIPTION
**Pull request by hishamco**
Fixes #59

### CODE REVIEW

**Comment by hishamco** [comment]
After trying different scenarios with `truncate` & ellipses using http://dotliquidmarkup.org/try-online, seems our previous needs to be change, we need to compare with ellipses length instead of the input length

/cc @sebastienros

File: Fluid/Filters/StringFilters.cs [hunk 1]
```diff
@@ -145,27 +145,34 @@ public static FluidValue Strip(FluidValue input, FilterArguments arguments, Temp
 
         public static FluidValue Truncate(FluidValue input, FilterArguments arguments, TemplateContext context)
         {
-            string text = input.ToStringValue();
-            int length = Convert.ToInt32(arguments.At(0).ToNumberValue());
+            var ellipsis = "...";
```
Comment (sebastienros) [inline Fluid/Filters/StringFilters.cs]: `const` at the class level

File: Fluid.Tests/StringFiltersTests.cs [hunk 2]
```diff
@@ -199,7 +199,7 @@ public void Truncate()
         {
             var input = new StringValue("Hello World");
 
-            var arguments = new FilterArguments().Add(new NumberValue(5)).Add(new StringValue("..."));
+            var arguments = new FilterArguments().Add(new NumberValue(8));
```
Comment (sebastienros) [inline Fluid.Tests/StringFiltersTests.cs]: Can you refactor to a `[Theory]` with `input`, `output` and `size` to handle the edge cases?

File: Fluid/Filters/StringFilters.cs (hunk 1 above)
Comment (hishamco) [reply Fluid/Filters/StringFilters.cs]: I already did that at the beginning, but this will change at the future. We can add an ellipses as parameter

**Comment by sebastienros** [comment]
@hishamco I have added more edge cases and also fixed them. GTG when I get your approval.

File: Fluid.Benchmarks/Fluid.Benchmarks.csproj [hunk 3]
```diff
@@ -5,6 +5,12 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
```
Comment (hishamco) [inline Fluid.Benchmarks/Fluid.Benchmarks.csproj]: Is this needed?!!

Comment (sebastienros) [reply Fluid.Benchmarks/Fluid.Benchmarks.csproj]: Unrelated to your PR, but saw this folder shouldn't be included.

File: Fluid/Filters/StringFilters.cs [hunk 4]
```diff
@@ -1,12 +1,13 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
 using Fluid.Values;
 
 namespace Fluid.Filters
 {
     public static class StringFilters
     {
+        private static StringValue Ellipsis = new StringValue("...");
```
Comment (hishamco) [inline Fluid/Filters/StringFilters.cs]: `static readonly` is better in this case

**Comment by hishamco** [comment]
LGTM after my above comment
